src/tribler/core/utilities/async_group/async_group.py

```diff
@@ -3,38 +3,38 @@
 from contextlib import suppress
 from typing import Iterable, List, Set
 
+from tribler.core.utilities.async_group.exceptions import CancelledException
+
 
 class AsyncGroup:
     """This class is a little brother of TaskManager and his purpose is only to
     correct cancel a group asyncio Tasks/Futures
 
     Example:
-    >>> from tribler.core.utilities.async_group import AsyncGroup
+    >>> from tribler.core.utilities.async_group.async_group import AsyncGroup
     >>> async def void():
     ...     pass
     >>> group = AsyncGroup()
-    >>> group.add([
-    ...     void(),
-    ...     void(),
-    ...     void()
-    ... ])
+    >>> group.add_task(void())
+    >>> group.add_task(void())
+    >>> group.add_task(void())
     >>> await group.cancel()
     """
 
     def __init__(self):
         self._futures: Set[Future] = set()
+        self._cancelled = False
```

Comment:
**nitpick**: While both "canceled" and "cancelled" are acceptable for the past tense of cancel, the version with one L is more common in American English, while the version with two L's is more common in British English.

In Tribler code (as in programming in general), American English is used more often than British English. For example, we use "color" and not "colour", "center" and not "centre". (On the other side, we are currently using both "behavior" and "behaviour" with the same frequency).

For that reason, it may be better to use "canceled" instead of "cancelled", and "CanceledException" instead of "CancelledException".

Reply:
Implemented